amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -1951,6 +1951,91 @@ VX_API_CALL vx_node VX_API_CALL vxExtrppNode_Nop(vx_graph graph, vx_tensor pSrc,
     return node;
 }
 
+VX_API_ENTRY vx_node VX_API_CALL vxExtrppNode_NonSilentRegion(vx_graph graph, vx_tensor pSrc, vx_tensor pDst1, vx_tensor pDst2, vx_tensor srcDims, vx_scalar cutOffDB, vx_scalar referencePower, vx_scalar window_length, vx_scalar resetInterval, vx_uint32 nbatchSize)
+{
+    vx_node node = NULL;
+    vx_context context = vxGetContext((vx_reference)graph);
+    if (vxGetStatus((vx_reference)context) == VX_SUCCESS)
+    {
+        vx_uint32 dev_type = getGraphAffinity(graph);
+        vx_scalar DEV_TYPE = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &dev_type);
+        vx_scalar NBATCHSIZE = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &nbatchSize);
+        vx_reference params[] = {
+            (vx_reference)pSrc,
+            (vx_reference)pDst1,
+            (vx_reference)pDst2,
+            (vx_reference)srcDims,
+            (vx_reference)cutOffDB,
+            (vx_reference)referencePower,
+            (vx_reference)window_length,
+            (vx_reference)resetInterval,
+            (vx_reference)NBATCHSIZE,
+            (vx_reference)DEV_TYPE};
```

Comment:
Rename DEV_TYPE to devType 
Applicable for other API too

Reply:
done